website/docs/r/policy_l7_access_profile.html.markdown

```diff
@@ -0,0 +1,116 @@
+
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_l7_access_profile" "test" {
+  display_name          = "test_l7_access_profile"
+  description           = "Terraform provisioned L7AccessProfile"
+  default_action_logged = true
+  default_action        = "ALLOW"
+
+  l7_access_entry {
+    display_name = "entry1"
+    action       = "ALLOW"
+
+    attribute {
+      key              = "APP_ID"
+      values           = ["SSL"]
+      attribute_source = "SYSTEM"
+
+      sub_attribute {
+        key    = "TLS_CIPHER_SUITE"
+        values = ["TLS_RSA_EXPORT_WITH_RC4_40_MD5"]
+      }
+    }
+
+    disabled        = true
+    logged          = "true"
+    sequence_number = "100"
+  }
+
+  l7_access_entry {
+    display_name = "entry2"
+    action       = "ALLOW"
+
+    attribute {
+      key              = "URL_CATEGORY"
+      values           = ["Abused Drugs"]
+      attribute_source = "SYSTEM"
+    }
+    sequence_number = "200"
+  }
+
+  tag {
+    scope = "scope1"
+    tag   = "tag1"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `context` - (Optional) The context which the object belongs to
+    * `project_id` - (Required) The ID of the project which the object belongs to
+* `default_action` - (Optional) The action to be applied to all the services. Valid values are `ALLOW`, `REJECT`, `REJECT_WITH_RESPONSE`.
+* `default_action_logged` - (Optional) Flag to activate packet logging. Default is deactivated.
+* `l7_access_entry` - (Optional) Property containing L7 access entries for Policy L7 Access Profile.
+    * `nsx_id` - (Optional) The NSX ID of this L7 access entry. If set, this ID will be used to create the resource.
+    * `display_name` - (Required) Display name of the L7 access entry.
+    * `description` - (Optional) Description of the L7 access entry.
+    * `action` - (Optional) The action to be applied to all the services.  Valid values are `ALLOW`, `REJECT`, `REJECT_WITH_RESPONSE`.
+    * `attribute` - (Optional) Property containing attributes/sub-attributes for Policy L7 Access Profile.
+        * `attribute_source` - (Optional) Source of attribute value i.e whether system defined or custom value. Valid values are `SYSTEM`, `CUSTOM`.
+        * `custom_url_partial_match` - (Optional) True value for this flag will be treated as a partial match for custom url.
+        * `description` - (Optional) Description for attribute value.
+        * `key` - (Optional) URL_Reputation is currently not available. Please do not use it in Attribute Key while creating context profile.
```

Comment:
This is likely generated from the API spec, but it does not seem to make much sense.

Perhaps we might want to change it to something like

Key for attribute. Supported Attribute Keys are APP_ID, URL_CATEGORY, CUSTOM_URL

(we don't mention URL_REPUTATION since it's not supported)